src/sdl3/pixels.rs

```diff
@@ -226,15 +266,104 @@ pub struct PixelMasks {
     /// The alpha mask
     pub amask: u32,
 }
+/// Details about a pixel format, as returned by SDL_GetPixelFormatDetails.
+///
+/// This includes the format code, bits/bytes per pixel, channel masks,
+/// bit counts, and shift values for red, green, blue, and alpha.
+#[derive(Debug, Copy, Clone, Eq, PartialEq, Hash)]
+pub struct PixelFormatDetails {
+    /// The pixel format code
+    pub format: PixelFormat,
+    /// Bits per pixel (raw depth)
+    pub bits_per_pixel: u8,
+    /// Bytes per pixel (padded depth)
+    pub bytes_per_pixel: u8,
+    /// Red mask
+    pub rmask: u32,
```

Comment:
`r_mask` instead. Same for all others.
SDL code is camelCase/PascalCase oriented while Rust default is snake_case for variables and fields.